Add table azure_express_route_circuit. Closes #153 (#170)
turbot/steampipe-plugin-azure: docs/tables/azure_express_route_circuit.md

# Table: azure_express_route_circuit

An ExpressRoute circuit represents a logical connection between your on-premises infrastructure and Microsoft cloud services through a connectivity provider. You can order multiple ExpressRoute circuits. Each circuit can be in the same or different regions, and can be connected to your premises through different connectivity providers.

## Examples

### Basic info

```sql
select
  name,
  id,
  allow_classic_operations,
  circuit_provisioning_state
from
  azure_express_route_circuit;
```

> ### List express route circuits which have global reach enabled

Reviewer: ```suggestion
### List express route circuits with global reach enabled
```

Author: Updated

```sql
select
  name,
  sku_tier,
  sku_name
from
  azure_express_route_circuit
where
  global_reach_enabled;
```

> ### List premium category express route circuits

Reviewer: ```suggestion
### List premium express route circuits
```

# Table: azure_express_route_circuit

> ExpressRoute lets you extend your on-premises networks into the Microsoft cloud over a private connection with the help of a connectivity provider. With ExpressRoute, you can establish connections to Microsoft cloud services, such as Microsoft Azure and Microsoft 365.

Reviewer 2: ```suggestion
An ExpressRoute circuit represents a logical connection between your on-premises infrastructure and Microsoft cloud services through a connectivity provider. You can order multiple ExpressRoute circuits. Each circuit can be in the same or different regions, and can be connected to your premises through different connectivity providers.
```

## Examples

### Basic info

```sql
select
  name,
  id,
  allow_classic_operations,
  circuit_provisioning_state
from
  azure_express_route_circuit;
```

> 

Reviewer 2: ```suggestion
```

### List express route circuits which have global reach enabled

```sql
select
  name,
  sku_tier,
  sku_name
from
  azure_express_route_circuit
where
  global_reach_enabled;
```

> 

Reviewer 2: ```suggestion
```

> ### List the premium category express route circuits

Reviewer 2: ```suggestion
### List premium category express route circuits
```
